contrib/intel/jenkins/build.py

```diff
@@ -69,6 +69,30 @@ def build_fabtests(libfab_install_path, mode):
     common.run_command(['make', '-j32'])
     common.run_command(['make', 'install'])
 
+def extract_mpich(mpitype):
+
+    if (mpitype == 'mpich'):
+        src_dir = 'mpich'
+        dest = 'mpich_temp'
+        mpich_tar = 'mpich-4.1.tar.gz'
+    elif (mpitype == 'impi'):
+        src_dir = 'impi_mpichtest'
+        dest = 'impi_mpichtest'
+        mpich_tar = 'mpich-test.tar.gz'
+    else:
+        print(f"Invalid mpi type {mpitype}")
+        exit
+
+    cwd = os.getcwd()
+    if (os.path.exists(f'{cloudbees_config.build_dir}/{dest}') == True):
+        common.run_command(['rm','-rf', f'{cloudbees_config.build_dir}/{dest}'])
```

Comment:
can you use os.rmtree(f'{cloudbess_config.build_dir}/{dest}') to remove the directory instead of run_command?
That way it will match all the other calls in this function